taxi/urls.py

```diff
@@ -1,27 +1,23 @@
 from django.urls import path
 
 from .views import (
-    index,
-    CarListView,
-    CarDetailView,
-    DriverListView,
-    DriverDetailView,
-    ManufacturerListView,
-)
+    index, CarListView, CarDetailView,
+    DriverListView, DriverDetailView, ManufacturerListView)
 
 urlpatterns = [
     path("", index, name="index"),
-    path(
-        "manufacturers/",
-        ManufacturerListView.as_view(),
-        name="manufacturer-list",
-    ),
+    path("manufacturers/",
+         ManufacturerListView.as_view(),
+         name="manufacturer-list"
```

Comment:
According to the task requirements, URL names should use hyphens (`-`) instead of underscores (`_`). Please change `manufacturer-list` to `manufacturer-list` to follow this convention.